system/ee/ExpressionEngine/Core/Installer.php

```diff
@@ -23,6 +23,7 @@ public function boot()
         define('APPPATH', SYSPATH . 'ee/installer/');
         define('EE_APPPATH', BASEPATH);
 
+        define('PATH_PRO_ADDONS', SYSPATH . 'ee/ExpressionEngine/Addons/pro/levelups/');
```

Comment:
@intoeetive Are we still using the levelups directory for this?  Asking as I don't see this path in my local install (not 100% sure if it's part of the install process or something along those lines though).

Reply:
We don't use it, but the constant is still defined in other places - and was missing here